lib/reporters/base.js

```diff
@@ -276,6 +281,7 @@ function Base(runner) {
   }
   this.stats = runner.stats; // assigned so Reporters keep a closer reference
   this.runner = runner;
+  this.println = println;
```

Comment:
does this need to be an instance prop?  should it be a static prop?  if it was the latter, reporters wouldn't have to extend `Base` to use it.  the less `this`/`self` the better, imo

Reply:
Nope better as static. Done.